applications/dashboard/controllers/api/UsersApiController.php

```diff
@@ -57,16 +57,26 @@ public function __construct(
      * Delete a user.
      *
      * @param int $id The ID of the user.
+     * @param array $body The request body.
      * @throws NotFoundException if the user could not be found.
      */
-    public function delete($id) {
+    public function delete($id, array $body) {
         $this->permission('Garden.Users.Delete');
 
+        $in = $this->schema([
+            'deleteMethod:s?' => [
+                'description' => 'The deletion method / strategy.',
+                'enum' => ['keep', 'wipe', 'delete'],
+                'default' => 'delete',
+            ]
+        ], 'in');
+        $body = $in->validate($body);
+
         $in = $this->idParamSchema()->setDescription('Delete a user.');
         $out = $this->schema([], 'out');
```

Comment:
I think you'll be able to get this to properly display in the API docs if you juggle the order a little, similar to other endpoints.

```php
$this->idParamSchema()->setDescription('Delete a user.');
$in = $this->schema([
    'deleteMethod:s?' => [
        'description' => 'The deletion method / strategy.',
        'enum' => ['keep', 'wipe', 'delete'],
        'default' => 'delete',
    ]
], 'in');
$out = $this->schema([], 'out');

$body = $in->validate($body);
```